samples/exposed-ktor/gradle.properties

```diff
@@ -2,5 +2,5 @@ ktor_version=2.3.1
 kotlin_version=1.8.21
 logback_version=1.2.11
 kotlin.code.style=official
-exposed_version=0.41.1
+exposed_version=0.42.0
 h2_version=2.1.214
```

Comment:
Ran the sample to check if it works and it fails. But it also fails with the older version because the version variable names in `build.gradle.kts` don't match with these names. Switching the names in `gradle.properties` to camelCase allowed the sample to run (with both versions).

Could you confirm if you're getting the same error with either version and, if so, fix the names too?
```
Cannot get non-null property 'ktorVersion' on root project 'exposed-ktor-crud' as it does not exist
```

Reply:
Thank you.